docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.8"
+
+services:
+
+  db:
+    image: postgres:13-alpine
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_USER=postgres
```

Comment:
This is not secure. Please generate a non-default user/password combination and place it inside .env file. Then you can add 
```
    env_file:
      -  .env
 ```
 And then all of those will be pulled. Don't push your .env file! Just provide an example .env.example with all the parameters you use, but with different values. E.g. defaults will work in this case.

Reply:
I did.